### ISSUE TITLE
Reimplement it in Go

### ISSUE THREAD

**Issue by malkusch** [EnterpriseQualityCoding/FizzBuzzEnterpriseEdition#256]
Let's take it to the next level and make it web scale!

**Comment by AndyWendt** [comment]
Only java has the right levels of ridiculous abstractions for such a satirical work.   :smile:

**Comment by hooph00p** [comment]
+1

**Comment by VoyTechnology** [comment]
After a prolonged debate with the management a decision was reached that such rewrite would not provide sufficient benefits to justify investing time, money and engineering resources. Certain engineers have also objected to writing in Go as its syntax is not verbose enough.

**Comment by emiln** [comment]
After a thorough discussion, our enterprise architects decided to stay with Java. Johnny, age 61 and our highest seniority architect, put it in simple terms which I'll relay here:

> We don't want no newfangled "Go" nonsense. What's wrong with good old Java? Why all this foolish hipster drivel? I've used Java since '95 and I ain't seen no [redacted] need for all this here concurrency, built-in package [redacted], and all that jazz. I think they can [redacted] right off with their [redacted] [redacted] […]

Sadly we've had to completely redact the remainder of his extensive explanation as it descended into somewhat profane language over the course of his 42 minute rant. Suffice to say that this is not a direction it seems we'll be able to pursue with Johnny in charge.

**Comment by looneym** [comment]
@emiln we need to reopen this - closing comment not clear. Can you please escalate this. Thanks.

**Comment by emiln** [comment]
I have created a P2 incident with our IBM support team. As per our SLA, they'll get back to me within 21 business days with an estimate of when they'll start estimating how soon they can look into the incident.

**Comment by MarkMenard** [comment]
Could you please edit the closing comment to bring it in line with the "Contributor Code of Conduct". I think this is sufficient cause to re-open this issue.

**Comment by looneym** [comment]
@MarkMenard good point. I also notice that @emiln hasn't provided an update from the IBM support team and it has been in excess of the 21 days outlined in the SLA. 

Given the potential operational impact of this I think it's fair to increase the priority of this issue to `On-222^47` . Going forward it would be helpful if we could devise a better workflow to catch these edge cases and prevent wasted time. 

Do you have access to the Kanban board where the new workflow proposals are outlined? If not I can add a proposal to propose that you be added there with a view to proposing a better workflow for the above afforementioned process deficiencies.

**Comment by gihad** [comment]
After months of search we were not able to find enough go developers, further more the performance of go has been shown to be inferior to modern [java](https://www.techempower.com/benchmarks/#section=data-r14&hw=ph&test=json).

**Comment by VoyTechnology** [comment]
The link you have provided cannot count as a valid benchmark. Framework implementation might be the limiting factor in a benchmark, and does not determine the performance of the language. Furthermore the FizzBussEnterpriseEdition project does not use a web framework.

With that said, there were multiple applicants with Go in their resume. Weirdly enough they have left the company after learning about the software stack. We are still investigating the cause.

**Comment by gihad** [comment]
We had all 10 Go developer available for hire in the world and we let them leave our pipeline? How did we miss this opportunity? As you said we definitely need to investigate the root cause. I suspect that Go became too mainstream for them, hence my suggestion to re-implement in Rust or Haskell in #309 

Coolness factor and hipster points are much more important than aligning tech with business goals. We should get 2 steps ahead and re-implement in [Brainfuck](en.wikipedia.org/wiki/Brainfuck)
